README.uwu.md

```diff
@@ -0,0 +1,236 @@
+# uwuifyy
+
+[![build](https://github.com/sgoudham/uwuifyy/actions/workflows/build.yml/badge.svg)](https://github.com/sgoudham/uwuifyy/actions/workflows/build.yml)
+[![crate.io](https://img.shields.io/crates/v/uwuifyy)](https://crates.io/crates/uwuifyy)
+[![downloads](https://img.shields.io/crates/d/uwuifyy)](https://crates.io/crates/uwuifyy)
+[![license](https://img.shields.io/github/license/sgoudham/uwuifyy)](LICENSE)
+
+> *cuddles you* A wobust, customizabwe, bwazingwy-fast, efficient and easy-to-use command winye appwication to uwu'ify youw text! 
+>
+![](logo/uwuifyy-logo.png)
+>
+> Wogo Cwedits: Jade Nyewson 
+
+## Tabwe O-Of Contents 
+
+- [uwuifyy](#uwuifyy)
+    * [About](#about)
+    * [Featuwes](#featuwes)
+    * [Instawwation](#instawwation )
+        * [Homebwew](#homebwew)
+        * [Binyawies](#binyawies)
+            + [Windows](#windows)
+            + [Winyux \ macOS](#winyux-\-macos)
+        * [Wust \ Cawgo](#wust-\-cawgo)
+    * [Usage](#usage)
+        + [Text I-Input to Text Output ](#text-i-input-to-text-output)
+        + [Text I-Input to F-Fiwe Output ](#text-i-input-to-f-fiwe-output)
+        + [Fiwe I-Input to F-Fiwe Output](#fiwe-i-input-to-f-fiwe-output) 
+    * [Benchmawks](#benchmawks) 
+        + [The Compwete Wowks of Wiwwiam Shakespeawe](#the-compwete-wowks-of-wiwwiam-shakespeawe)
+        + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k+-tokyo-2020\21-o-owympics-tweets) 
```

Comment:
This link doesn't work and should be fixed

Reply:
Ive been trying to fix it to no avail. I'm not sure why its not linking.

Reply:
@lukedev820 it should be:
```md
        + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k%2B-tokyo-2020\21-o-owympics-tweets) 
```

Reply:
> @lukedev820 it should be:
> 
> ```lisp
>         + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k%2B-tokyo-2020\21-o-owympics-tweets) 
> ```

Thank you im not the best at writing markdown so the help if much appreciated!

Reply:
I just tried the change @nekowinston but it still doesn't work.

Reply:
Hang on committing a fix now!